rrmpg/models/cemaneigehystgr4j.py

```diff
@@ -0,0 +1,701 @@
+# -*- coding: utf-8 -*-
+# This file is part of RRMPG.
+#
+# RRMPG is free software with the aim to provide a playground for experiments
+# with hydrological rainfall-runoff-models while achieving competitive
+# performance results.
+#
+# You should have received a copy of the MIT License along with RRMPG. If not,
+# see <https://opensource.org/licenses/MIT>
+
+"""Interface to the Cemaneige + GR4J coupled hydrological model."""
+
+import numbers
+import numpy as np
+import pandas as pd
+import spotpy
+from spotpy.objectivefunctions import kge
```

Comment:
I tried running your tests locally and this library is not part of the current conda environment for RRMPG, hence the tests failed. Is there any reason why you prefer spotpy over the kge implementation in utils/metrics.py? I would prefer to keep a minimal amount of dependencies, since every dependency can potentially break the library. At least the tests pass if I simply replace the imported kge function from spotpy with the calc_kge function of utils/calc_kge.py